packages/components/src/custom-select-control/types.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import type { FocusEventHandler, MouseEventHandler } from 'react';
+
+/**
+ * The legacy object structure for the options array.
```

Comment:
```suggestion
 * The object structure for the options array.
```

There are a couple other mentions of "legacy" in the description comments in this file.

Reply:
Good catch, done